src/Symfony/Bundle/SecurityBundle/Resources/config/security_authenticator.php

```diff
@@ -0,0 +1,159 @@
+use Symfony\Component\Security\Http\Authenticator\RememberMeAuthenticator;
+use Symfony\Component\Security\Http\Authenticator\RemoteUserAuthenticator;
+use Symfony\Component\Security\Http\Authenticator\X509Authenticator;
+use Symfony\Component\Security\Http\EventListener\CheckCredentialsListener;
+use Symfony\Component\Security\Http\EventListener\RememberMeListener;
+use Symfony\Component\Security\Http\EventListener\SessionStrategyListener;
+use Symfony\Component\Security\Http\EventListener\UserCheckerListener;
+use Symfony\Component\Security\Http\Firewall\AuthenticatorManagerListener;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+
+        // Manager
+        ->set('security.authenticator.manager', AuthenticatorManager::class)
+            ->abstract()
+            ->tag('monolog.logger', ['channel' => 'security'])
+            ->args([
+                [], // authenticators
+                service('security.token_storage'),
+                service('event_dispatcher'),
+                null, // provider key
+                service('logger')->nullOnInvalid(),
+                param('security.authentication.manager.erase_credentials'),
+            ])
+
+        ->set('security.authenticator.managers_locator', ServiceLocator::class)
+            ->args([[]])
+
+        ->set('security.user_authenticator', UserAuthenticator::class)
+            ->args([
+                service('security.firewall.map'),
+                service('security.authenticator.managers_locator'),
+                service('request_stack'),
+            ])
+        ->alias(UserAuthenticatorInterface::class, 'security.user_authenticator')
+
+        ->set('security.authentication.manager', NoopAuthenticationManager::class)
+        ->alias(AuthenticationManagerInterface::class, 'security.authentication.manager')
+
+        ->set('security.firewall.authenticator', AuthenticatorManagerListener::class)
+            ->abstract()
+            ->args([
+                null, // authenticator manager
+            ])
+
+        // Listeners
+        ->set('security.listener.check_authenticator_credentials', CheckCredentialsListener::class)
+            ->tag('kernel.event_subscriber')
+            ->args([
+               service('security.encoder_factory'),
+            ])
+
+        ->set('security.listener.user_checker', UserCheckerListener::class)
+            ->abstract()
+            ->args([
+                abstract_arg('user checker'),
+            ])
+
+        ->set('security.listener.session', SessionStrategyListener::class)
+            ->abstract()
+            ->args([
+                service('security.authentication.session_strategy'),
+            ])
+
+        ->set('security.listener.remember_me', RememberMeListener::class)
+            ->abstract()
+            ->tag('monolog.logger', ['channel' => 'security'])
+            ->args([
+                [], // remember me services
```

Comment:
RememberMeListener does not accept an array but RememberMeServicesInterface. so this value seems wrong (was already wrong before). probably should use an abstract_arg instead.